Consoles/Homebrew.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Homebrew
+description: Guides for Homebrewing various Consoles.
+---
+
+<h2>Guides<h2>
+
+### Nintendo
+
+[Wii Guide](https://wii.guide) - Guide made by the RiiConnect team to Homebrew the Wii.
+
+[Hacks.Guide](https://hacks.guide/) - Guides made by the Hacks.Guide team to Homebrew the 3ds, WiiU and PS Vita.
+
+[NH Switch Guide](https://nh-server.github.io/switch-guide/) - A Nintendo Switch guide. You may consider checking out this [GBATemp thread](https://gbatemp.net/threads/the-ultimate-list-of-mods-to-enter-rcm.502145/) to see other alternatives to enter RCM without soldering, but it is recommended to buy an RCM Jig.
+
+[Homebrew Guide](https://switch.homebrew.guide/) - Another Nintendo Switch Homebrew guide. This guide uses SDSetup, tool that is not recommended by some members, but other than that this guide is fine.
+
+### Sony
+
+[Vita Guide](https://vita.hacks.guide/) - Guide to Homebrew the PS Vita made by the Hacks.Guide team.
+
+### Other
+
+[ConsoleMods](https://consolemods.org/) - Community driven Wiki dedicated for modding.
```

Comment:
have u checked this site already? cuz the reason i didnt add it was i hadnt checked its legitamacy